modules/pipeline-manifest/bin/_backplane_mirror_index.sh

```diff
@@ -129,7 +132,16 @@ if [[ -z $SKIP_MIRROR ]]; then
 
   echo "klusterlet-operator-bundle tag:"
```

Comment:
Should we move this inside the conditional?

Reply:
The og code called it that way to start. I know it's a bit inefficient/redundant to call the mce tag twice like that but since it was specifically moddified and non-reliant on release I left it as is. I also left the klusterlet echo like that for logging purposes, so as to show when the bundle tag had an operation and when it was null. But I defer to you on what would be best here for sure